keymanapp/keyman
# fix(core): strip markers in `actions_update_app_context_nfu()`

Fixes #10605.

Includes unit test for marker stripping; unit tests for the module extensively refactored to ensure that we are testing the full context and not inadvertently missing markers.

# User Testing

* **TEST_HIEROGLYPHIC_WINDOWS**
  1. Install Hieroglyphic Keyboard from keyman.com into Keyman for Windows.
  2. Open Wordpad.
  3. Select the Hieroglyphic keyboard
  4. In wordpad, type: <kbd>d</kbd><kbd>i</kbd><kbd>spacebar</kbd><kbd>spacebar</kbd>
  5. Expected Output : 𓂞
  6. Type:<kbd>backspace</kbd>
  7. Expected Output: Empty
